### PR TITLE
Some documentation edits

### PR DESCRIPTION
**Pull request by Bob131**
Just fixing some typos and such

### CODE REVIEW

File: docs/vsgi/server/fastcgi.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ You can run the FastCGI example with lighttpd:
     lighttpd -D -f examples/fastcgi/lighttpd.conf
 
 Apache
-~~~~~~
+------
```
Comment (arteymix) [inline docs/vsgi/server/fastcgi.rst]: if you use this level of title, you should remove Deployment as well.

File: docs/getting-started.rst [hunk 2]
```diff
@@ -74,13 +74,13 @@ locations, so this wont be necessary.
...
-    valac --pkg valum-0.1 --vapidir=vapi
-          -X -I/usr/local/include/valum-0.1 -X -lvalum-0.1 # compiler options
+    valac --pkg valum --vapidir=vapi
+          -X -I/usr/local/include/valum -X -lvalum # compiler options
           src/app.vala
           -o build/app
 
     # if installed in default location /usr
-    valac --pkg valum-0.1 src/app.vala -o build/app
+    valac --pkg valum src/app.vala -o build/app
```
Comment (arteymix) [inline docs/getting-started.rst]: It is missing the `--vapidir` flag to actually work.

**Comment by arteymix** [comment]
You might have other edits, so I'll keep that open until you feel the docs is good enough ;)

**Comment by Bob131** [comment]
Ah, okay. Also, quick commit style q: Do you prefer to have multiple commits that can be squashed in a PR, or is amend+force push preferred?

**Comment by arteymix** [comment]
In this case, as you are improving the documentation, a single commit is sufficient.

More generally, each commit should focus on a particular feature or improvement. I typically squash fixes with `git rebase -i` on my local commits and push them when I feel they're good to go public. It's easier to revert particular feature this way and makes the history more understandable.

By the way, you manage this PR so feel free to rewrite the history as it suits you.

One last word, I keep two trunks: `master` and `next` that targets the next hotfix and minor release respectively. Features should be submitted on the latter.

**Comment by arteymix** [comment]
In fact I wonder if `next` is really relevant.. Feature submissions can just wait until the trunk moves to the  next minor release. Hotfixing a specific release can easily done by deriving its tag and releasing another tag.

**Comment by Bob131** [comment]
The version of the popular distilled wisdom on branch management in my head was that `master` should contain the latest code that builds and passes tests with tags for releases and branches for features. But if I'm expert in anything, it's not codebase management

**Pull request merged**